doctrinal-works-v3/original/44-chapter-44.adoc

```diff
@@ -16,6 +16,9 @@ that they must carry about them a body of death, and sins of the flesh, and impe
 whilst on this side the grave, which never did the prophets of God, Christ,
```

Comment:
I can't write on the exact line, but on line 4, do you want to make the [.heading-continuation-blurb] an [.inline] instead, to match all the other ones? I wasn't sure if I should change it or not.

Reply:
Yes, i think that would be better, but it involves one other step, so I'll do it after I merge. The rest looks great, thanks for working through all these chapters.  You did a great job!